test/js_test.go

```diff
@@ -962,17 +960,33 @@ func TestJetStreamImport(t *testing.T) {
 		defer os.RemoveAll(config.StoreDir)
 	}
 
-	// Create a stream using the server directly.
-	acc, _ := s.LookupAccount("JS")
-	mset, err := acc.AddStream(&server.StreamConfig{
+	// Create a stream using JSM.
+	u, err := url.Parse(s.ClientURL())
+	if err != nil {
+		t.Fatal(err)
+	}
+	authURL := fmt.Sprintf("nats://%s@%s", "dlc:foo", u.Host)
```

Comment:
`nats.Connect(s.ClientURL(), nats.User("dlc", "foo"))`

Reply:
Thanks, updated and build is green now.